managed/services/inventory/inventory_metrics.go

```diff
@@ -0,0 +1,245 @@
+// Copyright (C) 2017 Percona LLC
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+package inventory
+
+import (
+	"context"
+	"time"
+
+	"github.com/AlekSi/pointer"
+	prom "github.com/prometheus/client_golang/prometheus"
+	"gopkg.in/reform.v1"
+
+	"github.com/percona/pmm/api/inventorypb"
+	"github.com/percona/pmm/managed/models"
+	"github.com/percona/pmm/managed/utils/logger"
+)
+
+const (
+	requestTimeout              = 3 * time.Second
+	serviceEnabled      float64 = 1
+	prometheusNamespace         = "pmm_managed"
+	prometheusSubsystem         = "inventory"
+)
+
+type Metric struct {
+	labels []string
+	value  float64
+}
+
+//goland:noinspection GoNameStartsWithPackageName
+type InventoryMetrics struct {
+	db       *reform.DB
+	registry agentsRegistry
+}
+
+//goland:noinspection GoNameStartsWithPackageName
+type InventoryMetricsCollector struct {
+	mAgentsDesc   *prom.Desc
+	mNodesDesc    *prom.Desc
+	mServicesDesc *prom.Desc
+
+	metrics inventoryMetrics
+}
+
+func NewInventoryMetrics(db *reform.DB, registry agentsRegistry) *InventoryMetrics {
+	return &InventoryMetrics{
+		db:       db,
+		registry: registry,
+	}
+}
+
+func NewInventoryMetricsCollector(metrics inventoryMetrics) *InventoryMetricsCollector {
+	return &InventoryMetricsCollector{
+		mAgentsDesc: prom.NewDesc(
+			prom.BuildFQName(prometheusNamespace, prometheusSubsystem, "agents"),
+			"Inventory Agent",
```

Comment:
is there any reason why we can't expose the `node_name` label here (and in services metrics too)?